src/patterns.yml

```
@@ -3,112 +3,184 @@
+- "<quelquechose> : introduction à <quelquechose>"
 - "RGPD : <start>"
 - "WCAG 2.1 : <start>"
 - bien choisir <quelquechose_singulier>
 - trucs et astuces pour <verbe>
 - <quelquechose_pluriel_feminin> sont elles <adjectif_feminin_pluriel> ?
-- le charme <de_quelquechose> (fin)
-- revue <de_quelquechose>
+- le charme de <quelquechose> (fin)
+- revue de <quelquechose>
 - demain, <quelquechose>
-- le secret <de_quelquechose>
+- le secret de <quelquechose>
 - souriez, <quelquechose_pluriel> deviennent réalité
 - rendez-vous avec <quelquechose>
-- l'épopée <de_quelquechose> (quelquepart)
-- <quelquechose> de demain dans le web d'aujourd'hui (fin)
+- l’épopée de <quelquechose> (quelquepart)
+- <quelquechose> de demain dans le web d’aujourd’hui (fin)
 - concevoir <quelquechose> <fin>
-- guide de survie <de_quelquechose> (fin)
-- 3 méthodes pour <verbe> (quelquepart)
-- anatomie <de_quelquechose> <fin>
-- retour d'expérience sur <quelquechose>
+- guide de survie de <quelquechose> (fin)
+- 3 méthodes pour <verbe>
+- anatomie de <quelquechose> <fin>
+- retour d’expérience sur <quelquechose>
 - <quelquechose>, <quelquechose> et vous
-- ne passons pas à côté <de_quelquechose>
-- questionnements sur <quelquechose> <fin>
 - découvrir <quelquechose> par <quelquechose>
-- pourquoi <quelquechose_singulier> devrait s'intéresser à <le_quelquechose>
-- les coulisses <de_quelquechose> <quelquepart>
 - peut-on réellement <verbe> ?
 - y a-t-il de la place pour <verbe> <quelquepart> ?
-- vers l'infini et au delà <de_quelquechose>
-- <quelquechose> pour reprendre le contrôle <de_quelquechose>
+- vers l’infini et au delà de <quelquechose>
+- <quelquechose>, reprendre le contrôle de <quelquechose>
 - "<quelquechose> : comment <verbe>"
+- ne passons pas à côté de <quelquechose>
+- questionnements sur <quelquechose> <fin>
+- pourquoi <quelquechose_singulier> devrait s’intéresser à <quelquechose>
+- les coulisses de <quelquechose> <quelquepart>
+- vers l’infini et au delà de <quelquechose>
+- <quelquechose>, reprendre le contrôle de <quelquechose>
+- <quelquechose>, un modèle transversal pour <quelquechose>
+- <quelquechose> 2.0
+- vers une bonne pratique de <quelquechose>
+- "découper <quelquechose> : Pourquoi ? Comment ?"
+- <quelquechose>, <quelquechose>, <quelquechose>, faut-il réinventer <quelquechose_singulier> pour éviter la dystopie ?
+- le jour où l’on n’aura plus besoin de <quelquechose>
+- "évaluer <quelquechose> : des méthodes simples et efficaces"
+- "<quelquechose_pluriel> : vers de nouveaux usages dans <quelquechose_singulier> (plus_loin)"
+- <techno> fait pousser mes tomates (quelquepart)
+- <quelquechose> et comment on l’évite (fin)
+- créer un clone de <techno> en <techno> (fin)
+- votre site web est-il <adjectif_singulier>, <adjectif_singulier> et <adjectif_singulier> pour tous ?
+- "soyez <adjectif_pluriel> : construisez votre <techno> (fin)"
+- "<techno> / <techno> : bousculez vos habitudes !"
+- tester son <techno>, c’est possible !
```

```suggestion
- tester <techno>, c’est possible !
```

Pour éviter "son Blockchain" ?

---

j'ai retiré Blockchain, ça causait trop de problème (et il est déjà dans quelquechose)